src/status_im2/contexts/chat/messages/link_preview/events_test.cljs

```diff
@@ -0,0 +1,139 @@
+(ns status-im2.contexts.chat.messages.link-preview.events-test
+  (:require [cljs.test :as t :include-macros true]
```

Comment:
I think we don't need :include-macros anymore. And the testing macros are so ubiquitous in Clojure that it's fine to use `[cljs.test :refer [deftest testing is]]` as we usually do in tests. This should be in our guidelines, I believe it's not there. I suspect this `require` line was generated automatically for you in Emacs, it does for me too.